.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: python-bsonjs
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      # have to manually install pymongo for 3.4 and 3.5 because otherwise it does not work
+      - name: install pymongo for 3.4 and 3.5
+        run: |
+          python -m pip install pymongo==3.11.0	
+        if: ${{ matrix.python-version == 3.4 || matrix.python-version == 3.5}} 
+      - name: Test with python
+        run: |
+          python -m pip install -e . -v
+          python setup.py install
```

Comment:
Is `python setup.py install` needed. I would expect not because we already installed it via pip.

Reply:
Done.